spark/src/main/scala/io/delta/sharing/spark/DeltaSharingSource.scala

```diff
@@ -229,16 +234,16 @@ case class DeltaSharingSource(
    * @param isStartingVersion - If true, will load fromVersion as a table snapshot(including files
    *                            from previous versions). If false, will only load files since
    *                            fromVersion.
-   * @param currentLatestVersion - The latest table version returned from the delta sharing server.
-   *                               This is used to insert an indexedFile for each version in the
-   *                               sortedFetchedFiles, in order to ensure the offset move beyond
-   *                               this version.
+   * @param endingVersionForQuery - The ending version used for the query.
+   *                                This is used to insert an indexedFile for each version in the
+   *                                sortedFetchedFiles, in order to ensure the offset move beyond
+   *                                this version.
```

Comment:
Perhaps mention that the caller must ensure that ending version must be <= latest table version else the rpc will fail.

Reply:
While, this is not decided by the user, it's decided by the program. I updated the comment.